### DISCUSSION TITLE
GitHub Login Issue: Facing 404 Error and Automatic Logout

### DISCUSSION THREAD

**Discussion by asifakram74** [community/community#120971] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When I log in to GitHub and try to access any repository, it shows a 404 error and automatically logs me out. I've tried many times to log in, and while I can initially see the data, clicking on any link redirects me to a 404 page. This issue isn't isolated to just me; I've checked with my colleague's ID, and they're experiencing the same problem. Please help me figure out this issue and advise on how to fix it.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @asifakram74 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
